config/webpack/common.js

```diff
@@ -221,26 +192,24 @@ const createBaseConfig = ({
     devServer: {
       host: process.env.HOSTNAME || 'localhost',
       port: 3000,
-      disableHostCheck: true,
+      allowedHosts: 'all',
       hot: true,
-      publicPath: '/',
+      devMiddleware: {
+        publicPath: '/',
+      },
       historyApiFallback: true,
-      compress: true,
+      client: {
+        overlay: {
+          errors: true,
+          warnings: false,
+        },
+      },
     },
     performance: { hints: 'warning' },
-    // production optimizations
     optimization: {
       runtimeChunk: 'single',
-      minimizer: [
```

Comment:
Do we not need a minimizer anymore?

Reply:
webpack 5 comes with [terser builtin without any configuration necessary](https://webpack.js.org/plugins/terser-webpack-plugin/#:~:text=Webpack%20v5%20comes%20with%20the%20latest%20terser%2Dwebpack%2Dplugin%20out%20of%20the%20box.)